doctr/models/detection/core.py

```diff
@@ -145,12 +143,14 @@ def __call__(
         bitmap = tf.unstack(bitmap, axis=0)
 
         boxes_batch = []
+        # Kernel for opening, empirical law for ksize
+        k_size = 1 + int(p[0].shape[0] / 512)
```

Comment:
I think we only need to investigate whether this depends on the input size the model was trained on